portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -62,7 +65,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
    * Returns a list containing atmost maxComments top-level queries and all their replies. 
    * The top-level queries are sorted by sortMetric in sortOrder
    */ 
-  private void populateRootComments(ArrayList<UserComment> comments, int maxComments, String sortOrder, String sortMetric) {
+  private void populateRootComments(ArrayList<UserComment> comments, int maxComments
```

Comment:
If a method definition gets too long, the parameters are generally listed line-by-line. i.e.
```
private void populateRootComments(
    ArrayList<UserComment> comments, 
    int maxComments, 
    String sortOrder,
    String sortMetric, 
    String filterMetric, 
    String filterText) {
```